chain/store/name.go

```diff
@@ -117,30 +228,131 @@ func (cs *ChainStore) GetName(registrant []byte) (string, error) {
 	return cs.States.getName(registrant)
 }
 
+func (cs *ChainStore) GetRegName(name string, registrant string, expiresAt uint32) (string, error) {
```

Comment:
Same as above. Input a name for got a name is not make sense.

Reply:
done